### PR TITLE
feat(chartcuterie): Add devservice

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1521,6 +1523,11 @@ def create_partitioned_queues(name):
 SENTRY_USE_RELAY = True
 SENTRY_RELAY_PORT = 7899
 
+# Enables the chartcuterie service, which is used to render charts as images
+# for the Slack integration + sending emails.
+SENTRY_CHARTCUTERIE_ENABLED = False
```
Comment (EvanPurkhiser) [inline src/sentry/conf/server.py:1528]: Disabled by default.

**Review by armenzg — COMMENTED** [review]
I looked at it. Nothing scary but I don't understand enough to give it full approval.

File: src/sentry/conf/server.py [hunk 2]
```diff
@@ -1682,6 +1686,17 @@ def create_partitioned_queues(name):
         "only_if": lambda settings, options: settings.SENTRY_USE_RELAY,
         "with_devserver": True,
     },
+    "chartcuterie": {
+        "image": "us.gcr.io/sentryio/chartcuterie:nightly",
+        "pull": True,
+        "volumes": {CHARTCUTERIE_CONFIG_DIR: {"bind": "/etc/chartcuterie"}},
+        "environment": {
+            "CHARTCUTERIE_CONFIG": "/etc/chartcuterie/config.js",
```
Comment (EvanPurkhiser) [inline src/sentry/conf/server.py:1694]: This is where webpack puts this file

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -1682,6 +1686,17 @@ def create_partitioned_queues(name):
...
+            "CHARTCUTERIE_CONFIG_POLLING": "true",
+        },
+        "ports": {"9090/tcp": 7901},
+        "only_if": lambda settings, options: options.get("chart-rendering.enabled"),
```
Comment (wmak) [inline src/sentry/conf/server.py:1698]: nit: I feel like if this is only for the devservice I prefer using a setting instead of options but 🤷

**Review by wmak — APPROVED** [review]

File: src/sentry/conf/server.py (hunk 3 above)
Comment (EvanPurkhiser) [reply src/sentry/conf/server.py:1698]: This is not only for dev services. It will also be used to turn on and off this service as a whole.

Comment (wmak) [reply src/sentry/conf/server.py:1698]: Ahhh, makes sense :+1:

**Pull request merged**